pkgs/by-name/pd/pdfium-binaries/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash nixVersions.latest curl coreutils jq common-updater-scripts
+
+latestTag=$(curl -s ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} https://api.github.com/repos/bblanchon/pdfium-binaries/releases/latest | jq -r ".tag_name")
+latestVersion=$(echo $latestTag | sed 's/^chromium\///')
+currentVersion=$(nix-instantiate --eval -E "with import ./. {}; pdfium-binaries.version" | tr -d '"')
+
+echo "latest  version: $latestVersion"
+echo "current version: $currentVersion"
+
+if [[ "$latestVersion" == "$currentVersion" ]]; then
+    echo "package is up-to-date"
+    exit 0
+fi
+for i in \
+    "x86_64-linux linux-x64" \
+    "aarch64-linux linux-arm64" \
+    "x86_64-darwin mac-x64" \
+    "aarch64-darwin mac-arm64"; do
+    set -- $i
+    prefetch=$(nix-prefetch-url --unpack "https://github.com/bblanchon/pdfium-binaries/releases/download/chromium%2F$latestVersion/pdfium-$2.tgz")
+    hash=$(nix hash convert --hash-algo sha256 --to sri $prefetch)
```

Comment:
`nix hash convert` has been deprecated and removed in newer versions of Nix: use `nix hash to-sri --type sha256` instead